test/Microsoft.AspNet.Razor.Test/Framework/ParserTestBase.cs

```diff
@@ -277,15 +277,14 @@ public static void EvaluateParseTree(Block actualRoot, Block expectedRoot)
             // Evaluate the result
             var collector = new ErrorCollector();
 
-            // Link all the nodes
-            expectedRoot.LinkNodes();
-
             if (expectedRoot == null)
             {
                 Assert.Null(actualRoot);
             }
             else
             {
+                // Link all the nodes
+                expectedRoot.LinkNodes();
```

Comment:
Its totally unrelated to this change. But the code didn't make sense since the null check for `expectedRoot` comes after `expectedRoot.LinkNodes` method call.
Will add the blank line.